syft/workers/base.py

```diff
@@ -221,6 +225,27 @@ def registration_enabled(self):
         finally:
             self.is_client_worker = True
 
+    def connect_to_mpc(self, rank, world_size):
+        import os
+        default_args = {
+             "DISTRIBUTED_BACKEND": "gloo",
```

Comment:
Shouldn't this be BACKEND?

Reply:
I took this from the ```crypten/communicator/distributed_communicator.py``` and it also seems that on line 35 in the ```DistributedCommunicator``` the check is done for ```distributed_backend```.upper()